hermes-common/src/main/java/pl/allegro/tech/hermes/common/metric/HermesMetrics.java

```diff
@@ -273,5 +254,22 @@ public void reportConsumersWorkloadStats(String kafkaCluster, int missingResourc
     public Timer subscriptionLatencyTimer(Subscription subscription) {
         return timer(SUBSCRIPTION_LATENCY, subscription.getTopicName(), subscription.getName());
     }
+
+    public Timer oAuthProviderLatencyTimer(String oAuthProviderName) {
```

Comment:
maybe instead of adding another method here, we can simply call `hermesMetrics.timer("oauth-providers.<name>.latency")` ?